cleanrl/ppo2_continuous_action.py

```diff
@@ -300,6 +319,10 @@ def forward(self, x):
         values[step] = vf.forward(obs[step:step+1])
 
         with torch.no_grad():
+            # DEBUG: Tracking the logits
+            logits = pg(obs[step:step+1])
+            logits = logits[0][0].cpu().numpy()
+
```

Comment:
This shouldn't be here as this is in the `cleanrl` official folder. I do think they should be added to a separate file like in `cleanrl/experiments/ppo2_continuous_action_debug.py`